apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -4,7 +4,9 @@ import {
 } from "~/modules/users/libs/types/types.js";
 import { type UserService } from "~/modules/users/user.service.js";
 
-class AuthService {
+import { type AuthService as AuthServiceT } from "./libs/types/auth-service.type.js";
+
+class AuthService implements AuthServiceT {
```

Comment:
```suggestion
class AuthService {
```
No need to create AuthService type